jsonschema-validation.xml

```diff
@@ -536,84 +530,66 @@
                     <xref target="json-schema">core JSON Schema.</xref>
                     <cref>
                         Note that the "type" keyword in this specification defines an "integer" type
-                        which is not part of the data model.  Therefore a format attribute can be
-                        limited to numbers, but not specifically to integers.  However, a numeric
+                        which is not part of the data model. Therefore a format attribute can be
+                        limited to numbers, but not specifically to integers. However, a numeric
                         format can be used alongside the "type" keyword with a value of "integer",
                         or could be explicitly defined to always pass if the number is not an integer,
                         which produces essentially the same behavior as only applying to integers.
                     </cref>
                 </t>
 
                 <t>
-                    Meta-schemas that do not use "$vocabulary" SHOULD be considered to
-                    utilize this vocabulary as if its URI were present with a value of false.
-                    See the Implementation Requirements below for details.
+                    The current URI for this vocabulary, known as the Format-Annotation vocabulary, is:
+                    &lt;https://json-schema.org/draft/2020-11/vocab/format-annotation&gt;.  This vocabulary
+                    is required by this specification.
                 </t>
                 <t>
-                    The current URI for this vocabulary, known as the Format vocabulary, is:
-                    &lt;https://json-schema.org/draft/2020-11/vocab/format&gt;.
+                    In addition to the Format-Annotation vocabulary, a secondary vocabulary is available
+                    for custom meta-schemas that defines "format" as an assertion. The URI for the
+                    Format-Assertion vocabulary, is:
+                    &lt;https://json-schema.org/draft/2020-11/vocab/format-assertion&gt;.
                 </t>
                 <t>
                     The current URI for the corresponding meta-schema is:
-                    <eref target="https://json-schema.org/draft/2020-11/meta/format"/>.
+                    <eref target="https://json-schema.org/draft/2020-11/meta/format"/>. Because the
+                    syntactic requirements of "format" do not change between the annotation and assertion
+                    vocabularies, the meta-schema is shared between them.
```

Comment:
Can we do this, given that meta/* files contain a $vocabulary keyword which documents the $vocabulary URI used for this keyword?

Reply:
🤔 good call.  I didn't see that.  I'll amend this and split out the meta-schemas.

Reply:
Agree on the first part, but not the second.

Reply:
✔️